client/src/main/scala/io/casperlabs/client/configuration/Options.scala

```diff
@@ -109,6 +186,45 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
   }
   addSubcommand(deploy)
 
+  val sign = new Subcommand("sign") {
```

Comment:
```suggestion
  val sign = new Subcommand("sign-deploy") {
```

Just to fit in with the others?